#! /bin/sh /usr/share/dpatch/dpatch-run
## 900_ri_pager.dpatch by akira yamada <akira@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: try 'pager' before 'less'.
## DP: See http://redmine.ruby-lang.org/issues/show/1171

@DPATCH@
Index: b/lib/rdoc/ri/display.rb
===================================================================
--- a/lib/rdoc/ri/display.rb	2010-07-20 21:45:46.000000000 +0900
+++ b/lib/rdoc/ri/display.rb	2010-07-20 21:46:14.000000000 +0900
@@ -363,7 +363,7 @@
 
   def setup_pager
     unless @use_stdout then
-      for pager in [ ENV['PAGER'], "less", "more", 'pager' ].compact.uniq
+      for pager in [ ENV['PAGER'], "pager", "less", "more" ].compact.uniq
         return IO.popen(pager, "w") rescue nil
       end
       @use_stdout = true
